Zend/zend_compile.h

```diff
@@ -959,12 +955,13 @@ static zend_always_inline int zend_check_arg_send_type(const zend_function *zf,
 #define QUICK_ARG_MAY_BE_SENT_BY_REF(zf, arg_num) \
 	ZEND_CHECK_ARG_FLAG(zf, arg_num, ZEND_SEND_PREFER_REF)
 
-#define ZEND_RETURN_VAL 0
-#define ZEND_RETURN_REF 1
+#define ZEND_RETURN_VALUE				0
+#define ZEND_RETURN_REFERENCE			1
```

Comment:
Before I do/revert spacing issues, question of best-practice for PR's.  Interactive rebase, and remove the changes from prior commits, and force-update.  Or, a commit to revert the spacing changes, and then re-commit spacing changes after?